lib/css/walkCssTokens.js

```diff
@@ -301,21 +298,10 @@ const consumePotentialUrl = (input, pos, callbacks) => {
 			cc = input.charCodeAt(pos);
 		}
 		if (cc === CC_QUOTATION_MARK || cc === CC_APOSTROPHE) {
-			pos++;
-			const contentStart = pos;
-			pos = _consumeString(input, pos, cc);
-			const contentEnd = pos - 1;
-			cc = input.charCodeAt(pos);
-			while (_isWhiteSpace(cc)) {
-				pos++;
-				if (pos === input.length) return pos;
-				cc = input.charCodeAt(pos);
+			if (callbacks.function !== undefined) {
+				return callbacks.function(input, start, nextPos);
 			}
-			if (cc !== CC_RIGHT_PARENTHESIS) return pos;
-			pos++;
-			if (callbacks.url !== undefined)
-				return callbacks.url(input, start, pos, contentStart, contentEnd);
-			return pos;
+			return nextPos;
```

Comment:
Don't parse `url("img.png")` as `url` token, it should be parser as `Function` token as in CSS spec

Reply:
Can you add a comment here for this also.

Reply:
Ahhh okay. If we have bug # tied to PR that is fine.